packages/e2e-tests/run-e2e.sh

```diff
@@ -26,4 +26,5 @@ kill $wallet
 kill $app
 kill $hub
 killall node
+rm -rf ../../.ganache-deployments
```

Comment:
Add a check to see if the file exists? Otherwise this may fail.

Reply:
https://github.com/statechannels/monorepo/pull/1584/commits/7fe4f3416f9130a73302f437acc2c70f90e12a28